middlewares.ts

```diff
@@ -23,47 +23,63 @@ export const authorizePls = async (
     res: Response,
     next: NextFunction
 ) => {
-    const authorizationHeader = req.headers.authorization;
-    let token;
+    const authorizationHeader: string | undefined = req.headers.authorization;
+    let token = "";
+    let response: AxiosResponse | undefined = undefined;
```

Comment:
I do not like that you reassign response to use it for both http responses and errors

Reply:
Ok, semantically that was a bad idea, so I changed this now.